packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -1067,7 +1067,11 @@ class StackViewport extends Viewport implements IStackViewport {
     // TODO: this function will need to have more logic later
     // see http://dicom.nema.org/medical/Dicom/current/output/chtml/part03/sect_C.7.6.3.html#sect_C.7.6.3.1.2
     let numberOfComponents = 1;
-    if (photometricInterpretation === 'RGB') {
+    if (
+      photometricInterpretation === 'RGB' ||
+      photometricInterpretation === 'YBR_FULL' ||
```

Comment:
photometricInterpretation.indexOf('YBR')!=-1 would be better - there are 5-6 YBR versions